tsconfig.json

```diff
@@ -4,10 +4,10 @@
     "module": "CommonJS",
     "target": "es2015",
     "esModuleInterop": true,
-    "lib": ["esnext"],
+    "lib": ["esnext", "dom"],
```

Comment:
is this change actually necessary?

Reply:
yeah, Typescript is complaining direct access of `window`, even if we are doing `typeof window !== 'undefined`